core/lib/url-shim.js

```diff
@@ -270,5 +270,4 @@ URLShim.INVALID_URL_DEBUG_STRING =
     'Lighthouse was unable to determine the URL of some script executions. ' +
     'It\'s possible a Chrome extension or other eval\'d code is the source.';
 
-// TODO(esmodules): don't use default export
 export default URLShim;
```

Comment:
Maybe this should be a named export and used as URLShim everywhere (to denote it isn't the global node URL)? atm we just kinda shadow the global value in all the imports. so not 100% sure about ignoring this TODO.

Reply:
If not 100% sure we could just leave it as a normal `TODO` but then we will probably never get to it 🤷

Reply:
#14182 accidentally deleted the comment here about "remove this eventually". should be fine to remove now, and then make URLShim be called literally anything else. Could we maybe even remove the extending from URL and rename the things url-utils (and export its methods)?

Reply:
actually this needs to change to use global URL https://github.com/googleads/publisher-ads-lighthouse-plugin/blob/2687c2335942adb99ddeafcf6f5e3f92a8079f01/lighthouse-plugin-publisher-ads/audits/loads-gpt-from-official-source.js#L21


@brendankenny cc in case you know of more that needs to be done

Reply:
we could also just keep `export const URL = globalThis.URL;` as its pretty minimal... and there could be dependencies using `url` that are out of our control.